src/web/components/BusinessContacts/BusinessContact.tsx

```diff
@@ -84,7 +90,7 @@ function BusinessContact({
       <td>{contact.contactType}</td>
       <td className='action'>
         <div className='action-cell'>
-          {hasError && <InlineMessage message='An error has occurred' type='Error' />}
+          {hasError && <InlineMessage message={errorMessage} type='Error' />}
```

Comment:
I wonder if we still need `hasError` state? could we just use `!!errorMessage`?

Reply:
Good idea.  I changed it.